cmd/cli.go

```diff
@@ -13,15 +13,15 @@ const (
 	faucetPortUsage   = "The port on which to serve the faucet endpoint. Default: 80."
 
 	nodeHostName    = "nodeHost"
-	nodeHostDefault = "testnet.obscu.ro"
+	nodeHostDefault = "http://dev-testnet.obscu.ro"
```

Comment:
Sug: default to testnet.
```suggestion
	nodeHostDefault = "http://testnet.obscu.ro"
```